### PR TITLE
Script updates

### PR DESCRIPTION
**Pull request by Sjhunt93**

### CODE REVIEW

File: apiaudio/api_resources/script.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from ast import arg
```
Comment (martinezpl) [inline apiaudio/api_resources/script.py:1]: VS code 🤡

File: dev.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Dev notes
+
+To publish a new version, please run `bash publish.sh`.
```
Comment (martinezpl) [inline dev.md:3]: CI handles publishing and updates latest version checks in SDKs on main branch tag, avoid manual publishing

**Review by martinezpl — CHANGES_REQUESTED** [review]

File: dev.md (hunk 2 above)
Comment (Sjhunt93) [reply dev.md:3]: This was in the main readme? I just moved it. So just delete?

File: apiaudio/api_resources/script.py (hunk 1 above)
Comment (Sjhunt93) [reply apiaudio/api_resources/script.py:1]: oh its a pain - need more C++ and xcode :P

**Review by martinezpl — APPROVED** [review]
👍

File: README.md [hunk 3]
```diff
@@ -204,20 +210,23 @@ apiaudio.set_assume_org_id(None)
...
 
 There are two approaches to use the resources.
-First approach is to import the resource classes you want to use first, then use resource methods. For example, to use `Script`, we could do:
+
+The recommended approach is to import all resources directly from apiaudio:
 
 ```python
-from apiaudio import Script
-Script.create()
+import apiaudio
```
Comment (GetOnMyLvl97) [inline README.md:217]: much better

File: README.md [hunk 4]
```diff
@@ -229,20 +238,28 @@ Script methods are:
...
   - Example:
     ```python
+    text = """
+      <<sectionName::hello>> Hello {{username|buddy}} 
+      <<sectionName::bye>> Good bye from {{location|barcelona}}
+    """
+    
     script = apiaudio.Script.create(
-        scriptText="<<sectionName::hello>> Hello {{username|buddy}} <<sectionName::bye>> Good bye from {{location|barcelona}}",
+        scriptText=text,
```
Comment (GetOnMyLvl97) [inline README.md:255]: cleaner

File: dev.md (hunk 2 above)
Comment (martinezpl) [reply dev.md]: ah ok

**Review by GetOnMyLvl97 — APPROVED** [review]
lgtm

File: dev.md (hunk 2 above)
Comment (martinezpl) [reply dev.md]: yeah

**Review by springcoil — APPROVED** [review]
LGTM

**Pull request merged**